### PR TITLE
3X initial wiki

### PR DESCRIPTION
**Pull request by uzi**
Initial support for Wiki pages.  Probably more work to be done here, but it's a start (without being a supermegapatch).

👓 @schwers @nramadas

### CODE REVIEW

File: models2/Wiki.es6.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    mayRevise: T.bool,
+    revisionBy: T.nop,
+    revisionDate: T.number,
+  }
+
+  static API_ALIASES = {
+    content_html: 'contentHTML',
+    content_md: 'contentMD',
+    may_revise: 'mayRevise',
+    path: 'path',
```
Comment (schwers) [inline models2/Wiki.es6.js]: I would remove path here -- though this works I would keep aliases only as properties that renaming (read underscores to camelCase) here

**Comment by schwers** [comment]
💅  just a small nit on aliases

Nice catch on `es6.js`. This repo is yet to have the great `es6.js` -> `.js` migration ⬆️  for consistency in the interim

File: apis/wikis.es6.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import { runQuery } from '../apiBase/APIRequestUtils';
+import Wiki from '../models2/Wiki';
+
+const getPath = (query) => {
```
Comment (ajacksified) [inline apis/wikis.es6.js]: 🔕 we should get into the habit of `export`ing these functions and writing tests for them

Comment (uzi) [inline apis/wikis.es6.js]: 🔕 we should get into the habit of sending info and photos about new babies

**Comment by uzi** [comment]
💅

**Comment by uzi** [comment]
💇

File: apis/wikis.es6.js (hunk 2 above)
Comment (ajacksified) [inline apis/wikis.es6.js]: ssh don't tell my wife I'm on github 👶

**Comment by schwers** [comment]
:fish:

File: apis/wikis.es6.js (hunk 2 above)
Comment (schwers) [inline apis/wikis.es6.js]: ssh: Could not resolve hostname dont tell my wife Im: nodename nor servname provided, or not known

**Pull request merged**